<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyUdpOneWidget</class>
 <widget class="QWidget" name="MyUdpOneWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyUdpOneWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>对方IP</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="ipLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>对方端口</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="portLineEdit"/>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTextEdit" name="recvTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTextEdit" name="sendTextEdit"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="sendBtn">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>381</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="closeBtn">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
